src/components/attendee/SplashGated/SplashSpace.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect } from "react";
```

Comment:
there's a quite similar file `src/components/attendee/SplashSpace/SplashSpace.tsx`. do we need both?

Reply:
Deleted in https://github.com/sparkletown/sparkle/pull/3012/commits/2a4fe1dc4ef15783ba8a9a8fe44efb9b96c19540